keyboards/clawsome/bookerboard/readme.md

```diff
@@ -0,0 +1,13 @@
+/* Copyright 2020 AAClawson (AlisGraveNil)
```

Comment:
```suggestion
# Bookerboard
```

Reply:
Not trying to be contradictory, just curious: when I look through other folders of other kbs, I they all have a copyright header. What's wrong with mine?

Reply:
This is the readme, not a source file.

Reply:
ah